nextflow_schema.json

```diff
@@ -73,13 +73,13 @@
                 },
                 "min_val_gl": {
                     "type": "number",
-                    "description": "Minimum genotype likelihood probability P(G|R) in validation data. Set to zero to have no filter of if using –gt-validation",
+                    "description": "Minimum genotype likelihood probability P(G|R) in validation data. Set to zero to have no filter of if using \u2013gt-validation",
```

Comment:
Is there a reason to go for a unicode number for a simple dash ?

Reply:
I think this was the linter, thanks for spotting it!